route/src/route_generator_worker.cpp

```diff
@@ -339,17 +345,17 @@ namespace route {
     {
         if(this->rs_worker_.get_route_state() == RouteStateWorker::RouteState::FOLLOWING) {
             // convert from pose stamp into lanelet basic 2D point
-            lanelet::BasicPoint2d current_loc(msg->pose.position.x, msg->pose.position.y);
+            current_loc_(msg->pose.position.x, msg->pose.position.y);
```

Comment:
This is a bug. You cannot call a constructor on an existing object. This should be current_loc_ = lanelet::BasicPoint2d(msg->pose.position.x, msg->pose.position.y)